lisp/ess-r-mode.el

```diff
@@ -115,10 +115,17 @@
     (define-key ess-r-package-dev-map "s"    'ess-r-package-set-package)
     (define-key ess-r-package-dev-map "\C-a" 'ess-r-devtools-ask)
     (define-key ess-r-package-dev-map "a"    'ess-r-devtools-ask)
+    (define-key ess-r-package-dev-map "\C-e" 'ess-r-devtools-execute-command)
+    (define-key ess-r-package-dev-map "e"    'ess-r-devtools-execute-command)
+    (define-key ess-r-package-dev-map "\C-b" 'ess-r-devtools-build)
+    (define-key ess-r-package-dev-map "b"    'ess-r-devtools-build)
+    (define-key ess-r-package-dev-map "\C-B" 'ess-r-devtools-check-package-buildwin)
```

Comment:
So put it on \C-C then?

Reply:
I let you decide ;)

In my config I'm using `<leader> d c c` for normal checking and `<leader> d c w` for checking on Windows. Using an additional prefix key seems more extensible, i.e. we might want to add common r-hub platforms here.

Reply:
It makes sense but given that basic check is so much more common than all others I am a bit reluctant. If only R-hub worked as expected. I was never able to build on solaris there. 

Two potential alternatives:

  - put all "web" checks on C-c C-w C-w
  - offer alternative check options with completion on C-c C-w C 


BTW, shall we rename the command to `ess-r-package-check-with-winbuilder to avoid mnemonic confusion? The R-hub version could be `ess-r-package-check-with-rhub` with completion for OS types.

Reply:
> put all "web" checks on C-c C-w C-w

This seems reasonable.

>  shall we rename the command to ess-r-package-check-with-winbuilder to avoid mnemonic confusion?

Sounds good.

> with completion for OS types.

That'd be great! The completions can be generated with `rhub::platforms()`.

Reply:
My main use of rhub nowadays is the "ubuntu-rchk" platform btw. It runs Tomas's protection checks.